### PR TITLE
how-to/deploy, upgrade: update ansible-related deploy/upgrade docs

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

update ansible-related deploy/upgrade docs
### What is the related PR or file link(s)? 

To align with: https://github.com/pingcap/docs-cn/pull/1574, https://github.com/pingcap/docs-cn/pull/1677

### Which version does your change affect? 

2.1, 3.0, dev

### CODE REVIEW

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -481,12 +481,19 @@ location_labels = ["host"]
...
-1. For the cluster topology of multiple TiKV instances on each TiKV node, you need to edit the `block-cache-size` parameter in `tidb-ansible/conf/tikv.yml`:
+1. For the cluster topology of multiple TiKV instances on each TiKV node, you need to edit the `capacity` parameter under `block-cache-size` in `tidb-ansible/conf/tikv.yml`:
 
-    - `rocksdb defaultcf block-cache-size(GB)`: MEM \* 80% / TiKV instance number \* 30%
-    - `rocksdb writecf block-cache-size(GB)`: MEM \* 80% / TiKV instance number \* 45%
-    - `rocksdb lockcf block-cache-size(GB)`: MEM \* 80% / TiKV instance number \* 2.5% (128 MB at a minimum)
-    - `raftdb defaultcf block-cache-size(GB)`: MEM \* 80% / TiKV instance number \* 2.5% (128 MB at a minimum)
+    ```
+    storage:
+        block-cache:
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
      block-cache:
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -500,11 +507,16 @@ location_labels = ["host"]
...
 
-    Recommended configuration: `number of instances * parameter value = CPU_Vcores * 0.8`.
+    Recommended configuration: the number of TiKV instances \* the parameter value = the number of CPU cores \* 0.8.
 
 3. If multiple TiKV instances are deployed on a same physical disk, edit the `capacity` parameter in `conf/tikv.yml`:
 
-    - `capacity`: total disk capacity / number of TiKV instances (the unit is GB)
+    ```
+    raftstore:
+        capacity: 0
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]:
```suggestion
      capacity: 0
```

File: v2.1/how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -500,11 +500,16 @@ location_labels = ["host"]
...
 
-    Recommended configuration: `number of instances * parameter value = CPU_Vcores * 0.8`.
+    Recommended configuration: the number of TiKV instances \* the parameter value = CPU_Vcores \* 0.8.
 
 3. If multiple TiKV instances are deployed on a same physical disk, edit the `capacity` parameter in `conf/tikv.yml`:
 
-    - `capacity`: total disk capacity / number of TiKV instances (the unit is GB)
+    ```
+    raftstore:
+        capacity: 0
```
Comment (lilin90) [inline v2.1/how-to/deploy/orchestrated/ansible.md]:
```suggestion
      capacity: 0
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 4]
```diff
@@ -100,36 +104,56 @@ Pay special attention to the following variables configuration. For variable mea
...
-    # will be set to CPU_NUM * 0.8.
-    # high-concurrency: 8
-    # normal-concurrency: 8
-    # low-concurrency: 8
-```
+- In TiKV configuration, `end-point-concurrency` is changed to three parameters: `high-concurrency`, `normal-concurrency` and `low-concurrency`.
+
+    ```yaml
+    readpool:
+    coprocessor:
```
Comment (lilin90) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
      coprocessor:
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 5]
```diff
@@ -480,12 +480,19 @@ location_labels = ["host"]
 
 **Edit the parameters in the service configuration file:**
 
-1. For the cluster topology of multiple TiKV instances on each TiKV node, you need to edit the `block-cache-size` parameter in `tidb-ansible/conf/tikv.yml`:
+1. For the cluster topology of multiple TiKV instances on each TiKV node, you need to edit the `capacity` parameter under `block-cache-size` in `tidb-ansible/conf/tikv.yml`:
+
+    ```
+    storage:
+        block-cache:
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
      block-cache:
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 6]
```diff
@@ -499,11 +506,16 @@ location_labels = ["host"]
...
 
-    Recommended configuration: `number of instances * parameter value = CPU_Vcores * 0.8`.
+    Recommended configuration: the number of TiKV instances \* the parameter value = the number of CPU cores \* 0.8.
 
 3. If multiple TiKV instances are deployed on a same physical disk, edit the `capacity` parameter in `conf/tikv.yml`:
 
-    - `capacity`: total disk capacity / number of TiKV instances (the unit is GB)
+    ```
+    raftstore:
+        capacity: 0
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
      capacity: 0
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 7]
```diff
@@ -96,19 +101,39 @@ Edit the `inventory.ini` file. For IP information, see the `/home/tidb/tidb-ansi
 
 If you have previously customized the configuration file of TiDB cluster components, refer to the backup file to modify the corresponding configuration file in `/home/tidb/tidb-ansible/conf`.
 
-In TiKV configuration, `end-point-concurrency` is changed to three parameters: `high-concurrency`, `normal-concurrency` and `low-concurrency`.
+**Note for the following parameter changes:**
```
Comment (lilin90) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
**Note the following parameter changes:**
```

File: v2.1/how-to/upgrade/from-previous-version.md [hunk 8]
```diff
@@ -100,36 +104,56 @@ Pay special attention to the following variables configuration. For variable mea
 
 If you have previously customized the configuration file of TiDB cluster components, refer to the backup file to modify the corresponding configuration file in `/home/tidb/tidb-ansible/conf`.
 
-In TiKV configuration, `end-point-concurrency` is changed to three parameters: `high-concurrency`, `normal-concurrency` and `low-concurrency`.
+**Note for the following parameter changes:**
```
Comment (lilin90) [inline v2.1/how-to/upgrade/from-previous-version.md]:
```suggestion
**Note the following parameter changes:**
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 9]
```diff
@@ -96,19 +101,39 @@ Edit the `inventory.ini` file. For IP information, see the `/home/tidb/tidb-ansi
...
-    # will be set to CPU_NUM * 0.8.
-    # high-concurrency: 8
-    # normal-concurrency: 8
-    # low-concurrency: 8
-```
+- In TiKV configuration, `end-point-concurrency` is changed to three parameters: `high-concurrency`, `normal-concurrency` and `low-concurrency`.
+
+    ```yaml
+    readpool:
+    coprocessor:
```
Comment (lilin90) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
      coprocessor:
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 10]
```diff
@@ -122,22 +147,26 @@ $ ansible-playbook local_prepare.yml
...
-   ```
-  $ ansible-playbook excessive_rolling_update.yml
-   ```
+    ```shell
+    $ ansible-playbook excessive_rolling_update.yml
+    ```
+
+    > **Note:**
+    >
+    > If the current version is later than TiDB 3.0 versions, then you can still use the `rolling_update.yml` approach to rolling-update and rolling-restart the TiDB cluster.
```
Comment (lilin90) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
    > If the current version is TiDB 3.0.0 or is later than TiDB 3.0.0, you can still use the `rolling_update.yml` approach to rolling-update and rolling-restart the TiDB cluster.
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 11]
```diff
@@ -122,22 +147,26 @@ $ ansible-playbook local_prepare.yml
 
 > **Note:**
 >
-> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to  `PD service` under `systemd` in TiDB 3.0. Compared to earlier versions, rolling update operations on TiDB cluster components are slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
+> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to `PD service` under `systemd` in TiDB 3.0. If the current version is earlier than TiDB 3.0 versions, the rolling update of TiDB cluster components to the 3.0 versions is slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
```
Comment (lilin90) [inline v3.0/how-to/upgrade/from-previous-version.md]: Please also update `dev/how-to/upgrade/from-previous-version.md` accordingly.

Comment (lilin90) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to `PD service` under `systemd` in TiDB 3.0. If the current version is earlier than TiDB 3.0.0, the rolling update of TiDB cluster components to the 3.0 versions is slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
```

File: v3.0/how-to/upgrade/from-previous-version.md (hunk 10 above)
Comment (lilin90) [reply v3.0/how-to/upgrade/from-previous-version.md]: @superlzs0476 Is this what you mean in https://github.com/pingcap/docs-cn/pull/1677?

File: v3.0/how-to/upgrade/from-previous-version.md (hunk 11 above)
Comment (yikeke) [reply v3.0/how-to/upgrade/from-previous-version.md]: `dev/how-to/upgrade/from-previous-version.md` only provide two links. There is no such content in it.

Comment (lilin90) [reply v3.0/how-to/upgrade/from-previous-version.md]: Update that file according to changes [here](https://github.com/pingcap/docs-cn/pull/1574/files).

**Pull request merged**